.env.example

```diff
@@ -14,3 +14,9 @@ DIVIDER=1000000
 
 # make it 1 to be able to do some manual updates on the moc contracts
 TEST=0
+
+# Ether network to connect to when no wallet is connected
+DEFAULT_ETHER_NETWORK_ID=1
+DEFAULT_ETHER_NETWORK_NAME=mainnet
+# API key to use to connect to etherscan when no wallet is connected
+ETHERSCAN_API_KEY=TISS591URX3AHJGI5ZBW47I7KQIVEXTC79
```

Comment:
Is this the real etherscan api key? It's not really a secret, but still it shouldn't be in the example file.